### ISSUE TITLE
SeaMonkey Browser - Only Linux x64 - Won't start

### ISSUE THREAD

**Issue by anonimo82** [ptitSeb/box64#1798]
Hello everyone.
I'm trying to use Box86 to run a non game App from Linux, for Intel Arch (64 Bit) on Termux Proot Distro.
I'm getting this error:
`Error: PltResolver: Symbol  gdk_disable_multidevice(ver 0: gdk_disable_multidevice) not found, cannot apply R_X86_64_JUMP_SLOT 0x3f16b8b5c0 (0x3f109d5266) in /home/ivix/Downloads/seamonkey/libxul.so (local_maplib=(nil), global maplib=0x36c9fed0, deepbind=0)`

for this command line:
`box64 seamonkey`

Please, help.

Thanks in advance for any help. Regards, have a nice day.

**Comment by ptitSeb** [comment]
I have just added that function to box64, but there will probably be a lot of other function...

**Comment by anonimo82** [comment]
I don't understand, which function?

**Comment by ptitSeb** [comment]
> Error: PltResolver: Symbol gdk_disable_multidevice(ver 0: gdk_disable_multidevice) not found, cannot apply R_X86_64_JUMP_SLOT 0x3f16b8b5c0 (0x3f109d5266) in /home/ivix/Downloads/seamonkey/libxul.so (local_maplib=(nil), global maplib=0x36c9fed0, deepbind=0)

This function. I have added it to box64 wrapping so it now recognise it and will not show this error. If you update box64 to latest version, it should goes a bit farther.

**Comment by ptitSeb** [comment]
Yes, you have to wait, it's not updated yet as it's the same error.

**Comment by anonimo82** [comment]
Oh, ok. Now I feel quite dumb...

**Comment by LukeShortCloud** [comment]
@anonimo82 If you wanted to test it out, you'll need to build the latest Box64 code from source: https://github.com/ptitSeb/box64?tab=readme-ov-file#compilinginstallation
